storage/migrations/16_msig_transactions.go

```diff
@@ -0,0 +1,31 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 16 adds multisig transactions
+
+func init() {
+	up := batch(`
+	CREATE TABLE IF NOT EXISTS "multisig_transactions" (
+		"height" bigint not null,
+		"multisig_id" text not null,
+		"state_root" text not null,
+		"transaction_id" text not null,
+
+		"to" text not null,
+		"value" text not null,
+		"method" bigint not null,
+		"params" bytea not null,
+		"approved" json not null,
+		PRIMARY KEY ("height", "multisig_id", "state_root", "transaction_id")
```

Comment:
do you want state_root after multisig_id here?

Reply:
whoops, good catch